MemoryModule.c

```diff
@@ -645,6 +645,9 @@ HMEMORYMODULE MemoryLoadLibraryEx(const void *data, size_t size,
     } else {
         result->isRelocated = TRUE;
     }
+	
+    // flush instruction cache to avoid executing stale code after performing relocations
+    FlushInstructionCache((HANDLE)-1, NULL, 0);
```

Comment:
This should rather use `GetCurrentProcess()` instead of hardcoding `(HANDLE)-1)`.

Also it would probably be better to only flush the instruction cache of the actually modified sections (and only if relocations have been performed).